Access Control: Add fine-grained access control to ldap handlers (#35525)
grafana/grafana: docs/sources/enterprise/access-control/fine-grained-access-control-references.md

`fixed:users:org:read` | `org.users:read` | Allows to get user organizations.
`fixed:users:org:edit` | All permissions from `fixed:users:org:read` and <br>`org.users:add`<br>`org.users:remove`<br>`org.users.role:update` | Allows every read action for user organizations and in addition allows to administer user organizations.
`fixed:ldap:admin:read` | `ldap.user:read`<br>`ldap.status:read` | Allows to read LDAP information and status.

> `fixed:ldap:admin:edit` | All permissions from `fixed:ldap:admin:read` and <br>`ldap.user:sync`<br>`ldap.config:reload` | Allows every read action for LDAP and in addition allows to administer LDAP.

Reviewer: ```suggestion
`fixed:ldap:admin:edit` | All permissions from `fixed:ldap:admin:read` and <br>`ldap.user:sync`<br>`ldap.config:reload` | Read each action for LDAP, and administer LDAP.
```
What is the action here? Write a complete sentence, and include a period at the end if need be.

Reviewer: What does "Allows" mean here?

Author: That was there before i updated it. I only added <br>`ldap.config:reload`
